### DISCUSSION TITLE
Anyone running GraphRAG in AWS? If so, using what service stack?

### DISCUSSION THREAD

**Discussion by irl-dan** [microsoft/graphrag#553] [Q&A]
All of my other resources are in AWS, and I'm wondering if anyone runs this or an equivalent in AWS.

**Comment by richardhsuuuu** [comment]
not sure if there is anything standard yet.

**Comment by pratyush-chaudhary** [comment]
I am trying it right now, with Neptune + LlamaIndex, but hitting a lot of roadblocks :( 
LlamaIndex's abstractions seem to create a lot of issues to me.
